models/experimental/llama2_70b/tests/test_llama_mlp_t3000.py

```diff
@@ -8,25 +8,68 @@
 from models.utility_functions import get_devices_for_t3000, skip_for_grayskull
 from models.experimental.llama2_70b.tests.test_llama_mlp import run_test_LlamaMLP_inference
 
+import os
+
+# Set Llama flags for CI, if CI environment is setup
+if os.getenv("CI") == "true":
```

Comment:
Do you mind taking this opportunity to move this `os.getenv("CI") == "true"` logic that's used everywhere and put it into a utility function for tests?

Reply:
Will move them into a common function in next PR